classes/class-wc-rest-connect-stripe-settings-controller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Stripe_Settings_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Stripe_Settings_Controller extends WC_REST_Connect_Base_Controller {
+	protected $rest_base = 'connect/stripe/settings';
```

Comment:
Let's change this to `connect/stripe/oauth-url` or similar and rename the controller class to suit.